docs/configuration.md

```diff
@@ -249,6 +249,7 @@ An alert is an evaluated expression which can trigger actions like emailing or l
 * critNotification: comma-separated list of notifications to trigger on critical. This line may appear multiple times and duplicate notifications, which will be merged so only one of each notification is triggered. Lookup tables may be used when `lookup("table", "key")` is an entire `critNotification` value. See example below.
 * depends: expression that this alert depends on. If the expression is non-zero, this alert is unevaluated. Unevaluated alerts do not change state or become unknown.
 * ignoreUnknown: if present, will prevent alert from becoming unknown
+* unknownIsNormal: will convert unkown events into normal events. Useful in cases like elastic search where no data can be a good thing.
```

Comment:
Add that to the docs? Bet it will be a common question. Or, we can throw an error on the config if both exist.